TestAssets/TestProjects/TestAppWithLibrary/TestLibrary/project.json

```diff
@@ -10,9 +10,9 @@
     ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "NETStandard.Library": "1.5.0-rc3-24113-00"
   },
   "frameworks": {
-    "netstandard1.5": {}
+    "netstandard1.6": {}
```

Comment:
> Ok. So the right fix is to remove the --infer-runtimes options from our build scripts and add explicit runtimes sections in all our standalone netcoreapp1.0 projects.

That sounds like a good idea to me. `--infer-runtimes` is supposed to be removed sometime.  The less dependent we are on it, the better.

Make sure you only add `runtimes` for the standalone projects, not for libraries or for portable apps.